src/main/kotlin/sc/iview/Controls.kt

```diff
@@ -185,6 +184,9 @@ open class Controls(val sciview: SciView) {
         h.addKeyBinding("node: move selected one closer or further away", "ctrl scroll")
         h.addBehaviour("node: rotate selected one", NodeRotateControl(sciview))
         h.addKeyBinding("node: rotate selected one", "ctrl shift button1")
+        h.addBehaviour("node: delete selected one", ClickBehaviour { _, _ -> sciview.deleteActiveNode() })
```

Comment:
Do we want to warn the user? I think Blender throws up a dialog before deletion. This could be a call to the DeleteObject command and we could have that command contain the dialog warning?